## jupiter  安装好之后启动出现 权限问题：



``` shell
[E 13:59:42.431 NotebookApp] Failed to write server-info to /run/user/1000/jupyter/nbserver-35390.json: [Errno 13] Permission denied: '/run/user/1000/jupyter/nbserver-35390.json'
Traceback (most recent call last):
  File "/usr/local/bin/jupyter-notebook", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/jupyter_core/application.py", line 266, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/traitlets/config/application.py", line 658, in launch_instance
    app.start()
  File "/usr/local/lib/python3.6/dist-packages/notebook/notebookapp.py", line 1781, in start
    self.write_browser_open_file()
  File "/usr/local/lib/python3.6/dist-packages/notebook/notebookapp.py", line 1699, in write_browser_open_file
    with io.open(self.browser_open_file, 'w', encoding='utf-8') as f:
PermissionError: [Errno 13] Permission denied: '/run/user/1000/jupyter/nbserver-35390-open.html'
```

解决办法：

由于用户变量XDG_TUNTIME_DIR被设置导致问题，可以用

`unset XDG_RUNTIME_DIR`





**ref**: https://www.cnblogs.com/jodio/p/10400821.html

